crates/glaredb/src/args/mod.rs

```diff
@@ -49,6 +49,14 @@ pub struct RpcProxyArgs {
     /// Authorization code for communicating with Cloud.
     #[clap(long)]
     pub cloud_auth_code: String,
+
+    /// Path to server cert to use for TLS (required for TLS protocol).
+    #[clap(long)]
+    pub rpc_server_cert: Option<String>,
+
+    /// Path to server key to use for TLS (required for TLS protocol).
+    #[clap(long)]
+    pub rpc_server_key: Option<String>,
```

Comment:
could we have aliases for these, 
```sh
./glaredb proxy \
  --rpc-server-cert="path/to/cert" \
  --rpc-server-key="path/to/key"
 ``` 
 
 seems pretty long to me. 
 
 `--cert="cert"` and `--key="key"` seems pretty straightforward to me.